tests/cephfs/nfs/modifying_invalid_values_nfs_cluster.py

```diff
@@ -0,0 +1,201 @@
+        nfs_export_name = "/export_" + "".join(
+            secrets.choice(string.digits) for i in range(3)
+        )
+        export_path = "/"
+        fs_name = "cephfs"
+        if "5.0" in rhbuild:
+            client1.exec_command(
+                sudo=True,
+                cmd=f"ceph nfs export create cephfs {fs_name} {nfs_name} "
+                f"{nfs_export_name} path={export_path}",
+            )
+        else:
+            client1.exec_command(
+                sudo=True,
+                cmd=f"ceph nfs export create cephfs {nfs_name} "
+                f"{nfs_export_name} {fs_name} path={export_path}",
+            )
+
+        client1.exec_command(
+            sudo=True,
+            cmd=f"ceph nfs export get {nfs_name} {nfs_export_name} > export_original.conf",
+        )
+        out, rc = client1.exec_command(
+            sudo=True,
+            cmd="cat export_original.conf",
+        )
+
+        access_type_value = '"access_type": "RW"'
+        squash_value = '"squash": "none"'
+        security_value = '"security_label": true'
+
+        access_type = '"access_type": '
+        security_label = '"security_label": '
+        squash = '"squash": '
+
+        access_type_fault = "fault_value"
+        squash_fault = "fault_value"
+        security_fault = "fault_value"
+
+        log.info(f"config file for {nfs_export_name}: {out}")
+
+        client1.exec_command(sudo=True, cmd="cp export_original.conf export1.conf")
+        client1.exec_command(sudo=True, cmd="cp export_original.conf export2.conf")
+        client1.exec_command(sudo=True, cmd="cp export_original.conf export3.conf")
+        client1.exec_command(sudo=True, cmd="cp export_original.conf export4.conf")
+        client1.exec_command(sudo=True, cmd="cp export_original.conf export5.conf")
+        # changing access type to invalid value
+        client1.exec_command(
+            sudo=True,
+            cmd=f"sed -i 's/{access_type_value}/{access_type}{access_type_fault}/g' export1.conf",
+        )
+        out, rc = client1.exec_command(
+            sudo=True,
+            cmd="cat export1.conf",
+        )
+        log.info(f"config file for {nfs_export_name}: {out}")
+        log.info("Apply the config1")
+        out, ec = client1.exec_command(
+            sudo=True,
+            cmd=f"ceph nfs export apply {nfs_name} -i export1.conf",
+            check_ec=False,
+        )
+        if out:
+            raise CommandFailed("NFS export has permission to export1")
+        else:
+            log.info(ec)
+        log.info("changing squash to invalid value")
+        client1.exec_command(
+            sudo=True,
+            cmd=f"sed -i 's/{squash_value}/{squash}{squash_fault}/g' export2.conf",
+        )
+        out, rc = client1.exec_command(
+            sudo=True,
+            cmd="cat export2.conf",
+        )
+        log.info(f"config file for {nfs_export_name}: {out}")
+        log.info("Apply the config2")
```

Comment:
Can we log the conf file details as well Julius.. so that it will be easy to understand when there is a failure
out, rc = client1.exec_command(
            sudo=True,
            cmd="cat export.conf",
        )
        log.info(f"config file for {nfs_export_name}: {out}")
        
        At all the places for export1,export2...export5.conf

Reply:
it looks like I am printing all the modified export files... can u check again ?

Reply:
Thanks Julius,
my bad,It is printing.